### PR TITLE
Updates to survival notebook

### PR DESCRIPTION
**Pull request by sjspielman**
This PR requests to update the notebook [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1276) with several changes:

+ Don't use `*` or spaces in file names
+ Removed cancer groups with insufficient sample size. We need 3 samples _per_ living and deceased to be able to run the models, so craniopharyngioma and DMG have also been removed. This process of this is fully carried out in the notebook.
+ When possible we want to use the template functions in `util/survival_models.R` for overall module consistency
+ Wrote several wrapper functions to reduce repeated code from running similar survival models on different subsets of data

### CODE REVIEW

**Comment by sjspielman** [comment]
@jaclyn-taroni requesting a "review" here, thanks!

**Comment by jaclyn-taroni** [comment]
Nothing really major jumps out at me here. I have more minor comments, but we can have that conversation over in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1276 in my opinion.

File: analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd [hunk 1]
```diff
@@ -1,29 +1,27 @@
 ---
 title: "Survival analysis by TP53 and telomerase activity"
+authors: Run Jin (D3B), Jo Lynne Harenza (D3B), and Stephanie Spielman (CCDL)
```
Comment (jharenza) [inline analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd]:
```suggestion
authors: Run Jin (D3B), Jo Lynne Rokita (D3B), and Stephanie Spielman (CCDL)
```

**Review by jharenza — APPROVED** [review]
just updating my name and had a comment about whether you want to put functions in `util`, but going to approve so we can merge this

**Comment by sjspielman** [comment]
@jharenza good to merge!

**Pull request merged**